src/models/requestData.js

```diff
@@ -173,11 +140,16 @@ export default class RequestData {
     return rows.map(row => ({
       entryNumber: row.entry_number,
       hasErrors: row.issue_logs.filter(issue => issue.severity === 'error').length > 0,
-      columns: getVerboseColumns(row)
+      columns: getVerboseColumns(row, this.getColumnFieldLog())
     }))
   }
 
   getGeometries () {
+    if (!this.response || !this.response.details) {
+      logger.error('trying to get response details when there are none: request id: ' + this.id)
+      return undefined
+    }
+
     const geometryKey = this.getGeometryKey()
     const geometries = this.response.details.map(row => row.converted_row[geometryKey]).filter(geometry => geometry !== '')
```

Comment:
this line was a common cause of crashes, when trying to access the details of a bad request